src/libcprover-cpp/verification_result.h

```diff
@@ -80,4 +80,8 @@ struct verification_resultt
   std::unique_ptr<verification_result_implt> _impl;
 };
 
+// Allow translation of `verifier_resultt` into a CPROVER_EXIT_CODES (so that
```

Comment:
:pick: The use of a plural `CPROVER_EXIT_CODES` suggests one result is turned into multiple codes.
:thinking: Given that the API headers may be distributed without the rest of the project source, the reader maybe unaware of the existence of `src/util/exit_codes.h`. So maybe we should just call this a Command Line Interface (CLI) return value/exit code; especially given that the result is not strongly typed.